docs/source/guides/inference.rst

```diff
@@ -0,0 +1,57 @@
+.. _inference_documentation:
+
+Inference
+---------
+Anomalib provides entrypoint scripts for using a trained model to generate predictions from a source of image data. This guide explains how to run inference with the standard PyTorch model and the exported OpenVINO model.
+
+
+Torch Inference
```

Comment:
This needs to be updated to include `disable_show_images` and `visualization_mode`

Reply:
Thanks, good catch